doc/cephfs/fs-volumes.rst

```diff
@@ -46,33 +47,33 @@ Create a volume using::
     $ ceph fs volume create <vol_name> [<placement>]
 
 This creates a CephFS file system and its data and metadata pools. It can also
-try to create MDSes for the filesystem using the enabled ceph-mgr orchestrator
-module (see :doc:`/mgr/orchestrator`), e.g. rook.
+deploy MDS daemons for the filesystem using a ceph-mgr orchestrator
+module (see :doc:`/mgr/orchestrator`), e.g. Rook.
```

Comment:
s/e.g./for example/

This is just a suggestion, and as I make it I am aware that I contribute to the degradation of the English language. But people reading this can't be relied upon to know that "e.g." means "exempli gratia", and they can't be relied upon to know that "e.g." is not "i.e.". This isn't a showstopper. It's just my opinion.

Reply:
ack.  I swear we've s/eg./e.g./ elsewhere in the docs.  Pushing.